lib/absinthe/phase/document/execution/resolution.ex

```diff
@@ -256,11 +265,12 @@ defmodule Absinthe.Phase.Document.Execution.Resolution do
     schema.__absinthe_type__(schema_type)
   end
 
-  def error(node, message) do
+  def error(node, message, extra \\ []) do
```

Comment:
Possible change this to `%{}` if we store `extra` as a map, as I note in other comment.